src/components/ChartControls.tsx

```diff
@@ -22,7 +21,6 @@ export const RenderDefaultControls = ({ isMobile }: TRenderDefaultControls) => (
         <Timeperiod />
         <StudyLegend />
         <DrawTools />
-        <Views />
```

Comment:
ChartControls is called in Chart.tsx but 
we always pass chartControlsWidgets as null from deriv app. So it never get called.

```
 {chartControlsWidgets !== null && !enabledChartFooter && (
                                <ChartControls widgets={chartControlsWidgets} />
                            )}
```